test/RpcTest.hpp

```diff
@@ -13,7 +13,8 @@
 
 namespace Casper {
 
-inline std::string privKeyPemFile = std::filesystem::canonical("/proc/self/exe").parent_path().string() + "/data/cpp_sdk_test_secret_key.pem";
+inline std::string privKeyPemFile = std::filesystem::canonical("/proc/self/exe").parent_path().string() + "/data/test_source_secret_key.pem";
+inline std::string privKeyPemFileTarget = std::filesystem::canonical("/proc/self/exe").parent_path().string() + "/data/test_target_secret_key.pem";
```

Comment:
mhmm, we can not do this, because it won't work under macos... `"/proc/self/exe"` is linux specific. I tried in librpc to finding a specific file, which was needed by some tests, but it worked only when you triggered them manually and separately. If you wanted to trigger all tests via `ctest -VV -j 16`, then it didn't work because `ctest` started to search files in some totally different location than executable. And this behaviour is only for macos... At the moment I fixed it by copy content of file to string in separate files. You can check here: https://github.com/matterfi/libcsprrpc/pull/103